netmaster/master/endpointGroup.go

```diff
@@ -183,3 +181,35 @@ func DeleteEndpointGroup(tenantName, groupName string) error {
 
 	return docknet.DeleteDockNet(epgCfg.TenantName, epgCfg.NetworkName, epgCfg.GroupName)
 }
+
+//UpdateEndpointGroup updates the endpointgroups
+func UpdateEndpointGroup(bandwidth, key string, Dscp, burst int) error {
```

Comment:
Please follow Create/Delete egg example and take tenantName, groupName as arguments not egg key.
you can construct egg key from tenant, group names.